src/stories/atoms/Header/Header.tsx

```diff
@@ -24,9 +27,14 @@ const sizeTable: { [index: number]: string } = {
   [SizesEnum.Large]: '3rem',
 };
 
-const Wrapper = styled.div`
+const Wrapper = styled.div<{
+  screenSize: string;
+}>`
   display: flex;
-  flex-direction: column;
+  flex-direction: ${(props) =>
+    props.screenSize === screenSizes.Desktop ? 'row' : 'column'};
```

Comment:
👍

Reply:
This is just an experiment. I don't actually think displaying the heading text as row works for the app so I'll clean it up.